compliance_checker/cf/cf.py

```diff
@@ -1910,7 +1904,7 @@ def check_calendar(self, ds):
                                     valid_calendar,
                                     '§4.4.1 Time and calendar',
                                     reasoning)
-                ret_val.append(result)
+                    ret_val.append(result)
```

Comment:
While this does fix the code so it only appends an existing result, I think both the ret_val line and the result line should be un-indented, so there's always a Result object returned, not just when the calendar is invalid.

I know this goes a bit against the (possible) future philosophy of only report issues, but this would keep the check in line with the current philosophy. 

@lukecampbell please weigh in.